### PR TITLE
BZ 1579282 - Update access mode information

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1579282

### CODE REVIEW

File: architecture/additional_concepts/storage.adoc [hunk 1]
```diff
@@ -346,10 +348,9 @@ A volume's `AccessModes` are descriptors of the volume's capabilities. They
 are not enforced constraints. The storage provider is responsible for runtime
 errors resulting from invalid use of the resource.
 
-For example, a GCE Persistent Disk has `AccessModes` *ReadWriteOnce* and
-*ReadOnlyMany*. The user must mark their claims as `read-only` if they want to
-take advantage of the volume's ability for ROX. Errors in the provider show up
-at runtime as mount errors.
+For example, a GCE Persistent Disk has *ReadWriteOnce* access mode. You must
+mark the claims as `read-only` if you want to use the volume's
+ROX capability. Errors in the provider show up at runtime as mount errors.
```
Comment (liangxia) [inline architecture/additional_concepts/storage.adoc]: ROX is not suitable for GCE Persistent Disk. We may need to use Ceph or iSCSI for the example.

**Comment by tmorriso-rh** [comment]
@liangxia I changed the example from GCE to Ceph. Also, check Table 1 - do I need to change the supported access modes for GCE PD?

**Comment by liangxia** [comment]
> Also, check Table 1 - do I need to change the supported access modes for GCE PD?

Not sure what's the change you mean here.  

The following is what I see from the web.

Volume Plug-in | ReadWriteOnce | ReadOnlyMany | ReadWriteMany
-- | -- | -- | --
GCE Persistent Disk | ✅ | - | -

**Comment by liangxia** [comment]
The PR looks good to me now.

File: architecture/additional_concepts/storage.adoc [hunk 2]
```diff
@@ -29,10 +29,10 @@ additional PVCs. This has the effect of scoping a bound PV to a single
 xref:../../architecture/core_concepts/projects_and_users.adoc#namespaces[namespace]
 (that of the binding project).
 
-PVs are defined by a `PersistentVolume` API object, which represents a piece
-of existing, networked storage in the cluster that has been provisioned by an
-administrator. It is a resource in the cluster just like a node is a cluster
-resource. PVs are volume plug-ins like `Volumes`, but have a lifecycle
+PVs are defined by a `PersistentVolume` API object, which represents a piece of
+existing, networked storage in the cluster that has been provisioned by the
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: s/has been/was

File: architecture/additional_concepts/storage.adoc [hunk 3]
```diff
@@ -29,10 +29,10 @@ additional PVCs. This has the effect of scoping a bound PV to a single
...
+cluster administrator. It is a resource in the cluster just like a node is a
+cluster resource. PVs are volume plug-ins like `Volumes`, but have a lifecycle
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: no comma

File: architecture/additional_concepts/storage.adoc [hunk 4]
```diff
@@ -29,10 +29,10 @@ additional PVCs. This has the effect of scoping a bound PV to a single
...
 independent of any individual
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: s/independent/that is independent

File: architecture/additional_concepts/storage.adoc [hunk 5]
```diff
@@ -68,26 +68,26 @@ administrator configures one or more dynamic provisioners that provision storage
 and a matching PV.
 
 Alternatively, a cluster administrator can create a number of PVs in advance,
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: no comma

File: architecture/additional_concepts/storage.adoc [hunk 6]
```diff
@@ -68,26 +68,26 @@ administrator configures one or more dynamic provisioners that provision storage
...
-which carry the details of the real storage that is available for use by cluster
-users. PVs exist in the API and are available for consumption.
+which carry the details of the real storage that is available for use. PVs exist
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: s/which/that

File: architecture/additional_concepts/storage.adoc [hunk 7]
```diff
@@ -68,26 +68,26 @@ administrator configures one or more dynamic provisioners that provision storage
...
+in the API and are available for consumption.
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: s/consumption/use

File: architecture/additional_concepts/storage.adoc [hunk 8]
```diff
@@ -68,26 +68,26 @@ administrator configures one or more dynamic provisioners that provision storage
...
 [[binding]]
 
 === Binding
 
-A user creates a `PersistentVolumeClaim` with a specific amount of storage
-requested and with certain access modes and optionally a `StorageClass`. A
-control loop in the master watches for new PVCs. It either finds a matching PV
-or waits for a provisioner for the `StorageClass` to create one, then binds them
-together.
+Create a PVC with a specific amount of storage requested and with certain access
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: Maybe 
When you create a PVC, you request a specific amount of storage, specify the required access mode, and can create a StorageClass to describe and classify the storage. The control loop in the master watches for new PVCs and binds the new PVC to an appropriate PV. If an appropriate PV does not exist, a provisioner for the StorageClass creates one.

File: architecture/additional_concepts/storage.adoc [hunk 9]
```diff
@@ -68,26 +68,26 @@ administrator configures one or more dynamic provisioners that provision storage
...
+modes. Optionally, you can also create a storage class to describe and classify
+the storage. A control loop in the master watches for new PVCs. It either finds
+a matching PV or waits for a provisioner for the storage class to create one,
+then binds them together.
 
-The user will always get at least what they asked for, but the volume might be
-in excess of what was requested. This is especially true with manually
+You will always get the amount requested, but the volume might be
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: maybe s/You will always get the amount requested, but the volume might be in excess of your request./The PV volume might exceed your requested volume.

File: architecture/additional_concepts/storage.adoc [hunk 10]
```diff
@@ -98,12 +98,12 @@ with many manually provisioned 50Gi volumes would not match a PVC requesting
 
 Pods use claims as volumes. The cluster inspects the claim to find the bound
 volume and mounts that volume for a pod. For those volumes that support multiple
-access modes, the user specifies which mode is desired when using their claim as
+access modes, you must specify which mode is desired when using the claim as
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: s/which mode is desired when using/which mode applies when you use

File: architecture/additional_concepts/storage.adoc [hunk 11]
```diff
@@ -123,7 +123,9 @@ The purpose of PVC protection is to ensure that PVCs in active use by a pod are
 A PVC is in active use by a pod when the the pod status is `Pending`, and the pod is assigned to a node or the pod status is `Running`.
 ====
 
-When the PVC protection feature is enabled, if a user deletes a PVC in active use by a pod, the PVC is not immediately removed. PVC removal is postponed until the PVC is no longer actively used by any pods.
+When the PVC protection feature is enabled, if you delete a PVC in active use by
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: s/in/that is in

File: architecture/additional_concepts/storage.adoc [hunk 12]
```diff
@@ -304,7 +306,7 @@ modes describing that specific PV's capabilities.
 
 Claims are matched to volumes with similar access modes. The only two matching
 criteria are access modes and size. A claim's access modes represent a request.
-Therefore, the user may be granted more, but never less. For example, if a claim
+Therefore, you may be granted more, but never less. For example, if a claim
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: s/may/might

File: architecture/additional_concepts/storage.adoc [hunk 13]
```diff
@@ -346,10 +348,9 @@ A volume's `AccessModes` are descriptors of the volume's capabilities. They
 are not enforced constraints. The storage provider is responsible for runtime
 errors resulting from invalid use of the resource.
 
-For example, a GCE Persistent Disk has `AccessModes` *ReadWriteOnce* and
-*ReadOnlyMany*. The user must mark their claims as `read-only` if they want to
-take advantage of the volume's ability for ROX. Errors in the provider show up
-at runtime as mount errors.
+For example, Ceph has *ReadWriteOnce* access mode. You must
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: maybe s/has/offers or s/has/uses

File: architecture/additional_concepts/storage.adoc [hunk 14]
```diff
@@ -553,17 +554,17 @@ spec:
...
+class's name in the `storageClassName` attribute. Only PVs of the requested
+class, ones with the same `storageClassName` as the PVC, can be bound to the
+PVC. The cluster administrator can configure dynamic provisioners to service one
+or more storage classes. The cluster administrator can create a PV on demand
+that matches the specifications in the PVC.
 
-The cluster administrator can also set a default `StorageClass` for all PVCs.
+The cluster administrator can also set a default storage class for all PVCs.
 When a default storage class is configured, the PVC must explicitly ask for
 `StorageClass` or `storageClassName` annotations set to `""` to get bound to a
```
Comment (kalexand-rh) [inline architecture/additional_concepts/storage.adoc]: s/get/be

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of picks

**Pull request merged**